src/seedu/addressbook/data/AddressBook.java

```diff
@@ -54,6 +54,13 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Clears all persons and tags from the address book.
```

Comment:
It's good to have comments in the different edition of different classes so as to enhance the code readability.